src/Compilers/CSharp/Portable/Symbols/EventSymbol.cs

```diff
@@ -258,14 +261,16 @@ internal EventSymbol AsMember(NamedTypeSymbol newOwner)
         {
             Debug.Assert(this.IsDefinition);
             Debug.Assert(ReferenceEquals(newOwner.OriginalDefinition, this.ContainingSymbol.OriginalDefinition));
+#nullable disable // Can 'newOwner as SubstitutedNamedTypeSymbol' be null? https://github.com/dotnet/roslyn/issues/39166
```

Comment:
No, it cannot be.

Reply:
I'm planning to leave this one for a follow-up

Reply:
Why? The question has been answered? Same for all the other comments that I've answered in this PR.

---
In reply to: [368006450](https://github.com/dotnet/roslyn/pull/39194#discussion_r368006450) [](ancestors = 368006450)

Reply:
That was the original intent of all the `#nullable disable` lines in all the PRs I submitted. The comment was just added for clarity for whoever looks at these later during a cleanup pass.

Reply:
Sure, but I've taken the time and answered the questions, and I don't really want to leave these in a half-baked state.

Reply:
I'll get the ones in this pull request updated, but keep in mind for the ones that are still pending review?

Reply:
Alright.

Reply:
Updated